content/en/docs/concepts/overview/working-with-objects/_index.md

```diff
@@ -1,7 +1,137 @@
 ---
-title: "Working with Kubernetes Objects"
-weight: 40
+title: Objects In Kubernetes
+content_type: concept
+weight: 10
 description: >
-  Kubernetes objects are persistent entities in the Kubernetes system. Kubernetes uses these entities to represent the state of your cluster.
+  Kubernetes objects are persistent entities in the Kubernetes system.
+  Kubernetes uses these entities to represent the state of your cluster.
   Learn about the Kubernetes object model and how to work with these objects.
+simple_list: true
+card:
+  name: concepts
+  weight: 40
 ---
+
+<!-- overview -->
+
+This page explains how Kubernetes objects are represented in the Kubernetes API, and how you can
+express them in `.yaml` format.
+
+<!-- body -->
+
+## Understanding Kubernetes objects {#kubernetes-objects}
+
+*Kubernetes objects* are persistent entities in the Kubernetes system. Kubernetes uses these
+entities to represent the state of your cluster. Specifically, they can describe:
+
+* What containerized applications are running (and on which nodes)
+* The resources available to those applications
+* The policies around how those applications behave, such as restart policies, upgrades, and fault-tolerance
+
+A Kubernetes object is a "record of intent"--once you create the object, the Kubernetes system
+will constantly work to ensure that object exists. By creating an object, you're effectively
+telling the Kubernetes system what you want your cluster's workload to look like; this is your
+cluster's *desired state*.
```

Comment:
nit: possibly remove quotes around `record of intent`
nit: don't italicize `desired state`

Reply:
👍